Specs/Scene/CameraFlightPathSpec.js

```diff
@@ -108,6 +111,27 @@ describe(
       expect(camera.roll).toEqualEpsilon(endRoll, CesiumMath.EPSILON12);
     });
 
+    it("creates an animation in 3d using custom ellipsoid", function () {
+      var ellipsoid = new Ellipsoid(1737400, 1737400, 1737400);
+      var mapProjection = new GeographicProjection(ellipsoid);
+      scene = createScene({
+        mapProjection: mapProjection,
+      });
+      scene.globe = new Globe(ellipsoid);
+
+      var camera = scene.camera;
```

Comment:
for consistency with other animation specs here, could you please add a `var startPosition = Cartesian3.clone(camera.position)` and check that this code block from other tests passes - 
```
flight.update({ time: 0.0 });
expect(camera.position).toEqualEpsilon(
    startPosition,
    CesiumMath.EPSILON12
);
```
This way we know that both the start and end positions are as expected.